src/tests/Microsoft.PowerFx.Interpreter.Tests/InterpreterExpressionTestCases/ParallelAsync.txt

```diff
@@ -105,3 +105,9 @@ Table({N1:0,N2:1,Value:0},{N1:2,N2:3,Value:2},{N1:4,N2:5,Value:4})
 
 >> WaitFor(0); WaitFor(1); WaitFor(2)
 2
+
+>> WaitFor(0); 1; WaitFor(1)
+1
+
+>> WaitFor(0); 1/0 ; WaitFor(1)
```

Comment:
>>> [](http://example.com/codeflow?start=0&length=2)

nested test:

(W(0); ( W(1); W(2)) ) ;   W(3)

Reply:
Not supported